docs/partial_conf.py

```diff
@@ -48,3 +48,9 @@
 # That resolved a bug of autosummary generating docs for code-block examples
 # of autosummary
 autosummary_generate = ['index.rst']
+
+skippable_method_attributes = [
```

Comment:
Same for this one here.

Reply:
Yup, it's explained in the comment above it